next-env.d.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/triple-slash-reference */
```

Comment:
This is not necessary anymore since `xo` ESLint config was removed.

Reply:
Cool! That line kept getting removed when I ran Prettier, and I wasn't sure if we needed that line or not. Saves me a bit of hassle each commit. 😄